<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>equip</class>
 <widget class="QMainWindow" name="equip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>975</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>1161</width>
      <height>951</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
font-family: century gothic;
font-size:24px
}
*{
background:#333;
}
</string>
    </property>
    <property name="title">
     <string>le petit prince</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>951</width>
       <height>881</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="title">
      <string>Gestion des equipements</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Date d'arriv√©e</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Bloc</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>100</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>140</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>180</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>891</width>
        <height>601</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
background:#333;
border: 1px solid white ;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>611</width>
         <height>501</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#49ebff;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>90</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background:#49ebff ; 
border-radius:10px; 
color:#333; 
border: 2px solid gray ;
padding: 0.5px;
font-size: 12px;
}
QPushButton:hover
{
color:#333; 
border-radius:10px;
background:#c7cf00;
border: 2px solid gray ;
padding: 0.8px;
}</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>140</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background:#49ebff ; 
border-radius:10px; 
color:#333; 
border: 2px solid gray ;
padding: 0.8px;
font-size: 12px
}
QPushButton:hover
{
color:#333; 
border-radius:10px;
background:#c7cf00;
border: 2px solid gray ;
padding: 0.8px;
}</string>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>190</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background:#49ebff ; 
border-radius:10px; 
color:#333; 
border: 2px solid gray ;
padding: 0.8px;
font-size: 12px
}
QPushButton:hover
{
color:#333; 
border-radius:10px;
background:#c7cf00;
border: 2px solid gray ;
padding: 0.8px;
}</string>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>230</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background:#49ebff ; 
border-radius:10px; 
color:#333; 
border: 2px solid gray ;
padding: 0.8px;
font-size: 12px
}
QPushButton:hover
{
color:#333; 
border-radius:10px;
background:#c7cf00;
border: 2px solid gray ;
padding: 0.8px;
}</string>
       </property>
       <property name="text">
        <string>Recherche</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>230</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>280</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background:#49ebff ; 
border-radius:10px; 
color:#333; 
border: 2px solid gray ;
padding: 0.8px;
font-size: 12px
}
QPushButton:hover
{
color:#333; 
border-radius:10px;
background:#c7cf00;
border: 2px solid gray ;
padding: 0.8px;
}</string>
       </property>
       <property name="text">
        <string>Trie</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background:#49ebff ; 
border-radius:15px; 
color:#333; 
border: 2px solid gray ;
padding: 0.8px;
}
QPushButton:hover
{
color:#333; 
border-radius:15px;
background:#c7cf00;
border: 2px solid gray ;
padding: 0.8px;
}</string>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>120</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background:#49ebff ; 
border-radius:15px; 
color:#333; 
border: 2px solid gray ;
padding: 0.8px;
}
QPushButton:hover
{
color:#333; 
border-radius:15px;
background:#c7cf00;
border: 2px solid gray ;
padding: 0.8px;
}</string>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>groupBox</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>881</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>1123</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
